tests/benchmark.py

```diff
@@ -58,7 +58,12 @@ def run_benchmark_config(
         if output_save_dir:
             filename = (
                 f"{Path(input_path).stem}_"
-                + ", ".join([f"{key}={value}" for key, value in config.items()])
+                + ", ".join(
+                    [
```

Comment:
Not sure if `value` needs to be type `str` but was complaining for me
`f"{key}={str(value).replace('/', '_')}"`

Reply:
Oh, and how do you feel about adding/updating a `README.md` about extra steps on running benchmark?
`huggingface-cli login --token $HF_TOKEN --add-to-git-credential`

Reply:
Is the `huggingface-cli login` command you mentioned necessary? I don't recall doing it. I just specified the token in the `.env`.
Any other commands we need to mention?

Reply:
I think without `huggingface-cli login` it might throw an error but i might have to check again.
Let's skip it then, if it comes up again then we can consider adding?